test/test.js

```diff
@@ -1765,3 +1765,30 @@ test( "Validation triggered on radio and checkbox via click", function() {
 	// test if there is no error anymore
 	ok( form.find( "input.error" ).length === 0, "Form valid" );
 });
+
+test( "destruction of Validator", function() {
+	expect( 2 );
+
+	var form = $( "#validatorDestroy" ),
+		validate = form.validate();
+
+	form.on( "submit.validate", function( event )
+	{
```

Comment:
Unfortunately jscs is ignoring this - can you please stick to the existing style and remove the linebreak here?